### PR TITLE
Update buyer's premium copy and location

### PR DESCRIPTION
**Pull request by bhoggard**
Ticket: https://artsyproduct.atlassian.net/browse/AUCT-676

Screen Shot 2019-11-01 at 4 01 36 PM

### CODE REVIEW

File: src/Apps/Artwork/Components/ArtworkSidebar/__stories__/ArtworkSidebarCurrentBidInfo.story.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import {
   AuctionPreview,
   AuctionPreviewNoStartingBid,
   ClosedAuctionArtwork,
-  LiveAuctionInProgeress,
```
Comment (dleve123) [inline src/Apps/Artwork/Components/ArtworkSidebar/__stories__/ArtworkSidebarCurrentBidInfo.story.tsx]: 💯

File: src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo.tsx [hunk 2]
```diff
@@ -21,14 +21,6 @@ export class ArtworkSidebarAuctionPartnerInfo extends React.Component<
     subject: Schema.Subject.AuctionBuyerPremium,
     type: Schema.Type.Link,
   }))
-  onClickBuyerPremium(mediator) {
```
Comment (dleve123) [inline src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo.tsx]: Can we remove the query for the  `is_with_buyers_premium` field in the [graphql query for this component](https://github.com/artsy/reaction/pull/2966/files?utf8=%E2%9C%93&diff=unified&w=1#diff-c9ef16c3b323f531057622da216d4a53R52-R68)? I believe it's no longer needed now.

File: src/Apps/__tests__/Fixtures/Artwork/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.ts [hunk 3]
```diff
@@ -142,6 +142,8 @@ export const OpenAuctionReserveMetWithBids = {
   sale: {
     is_closed: false,
     is_live_open: false,
+    is_with_reserve: true,
+    is_with_buyers_premium: true,
```
Comment (dleve123) [inline src/Apps/__tests__/Fixtures/Artwork/ArtworkSidebar/ArtworkSidebarCurrentBidInfo.ts]: why this line of diff? seems unrelated

**Comment by artsyit** [comment]
:rocket: PR was released in v21.17.8 :rocket:
